[RHACS] Add known issues section to Release Notes (#67972)
openshift/openshift-docs: release_notes/43-release-notes.adoc

* In some cases, Common Vulnerabilities and Exposures (CVEs) that were deferred and approved were not added to the list of snoozed CVEs. This behavior was caused by an issue with the `/v1/suppress` and `/v1/unsuppress` APIs and has been fixed.
* Fixed file search in the `ca-setup.sh` file that is used to configure additional Certificate Authorities (CAs). Previously, the script could not find any files in the directory, even with the correct .crt and .pem extensions.

[id="known-issues-430_{context}"]
== Known issues

> * {product-title-short} version 4.2 introduced additional logic that considers the socket connection state to decide whether Collector should report a connection or not. The expected behavior is that the connection is not reported until it is successfully established. There is a known issue on the `ppc64le` architecture that a blocked or failing connection might be reported instead of being silenced. There is currently no workaround for this issue.

Reviewer: ```suggestion
* {product-title-short} version 4.2 introduced additional logic that considers the socket connection state to decide whether the Collector reports a connection or not. The expected behavior is that the connection is not reported until it is successfully established. A known issue on the `ppc64le` architecture can cause a blocked or failing connection to be reported instead of being silenced. No workaround exists for this issue.
```
- I'm not sure whether "Collector" takes an article.
- IBM Style advises against modals (should, would, could). For more information, see the [Word Usage page](https://www.ibm.com/docs/en/ibm-style?topic=word-usage) and search for the word "should".
- I reworded a couple of sentences to avoid starting with "There is", per IBM Style.

> * {product-title-short} Central components do not deploy correctly on a default {product-rosa-short} cluster. The workaround is to scale the worker nodes to allow the {product-title-short} components to schedule.

Reviewer: ```suggestion
* {product-title-short} Central components do not deploy correctly on a default {product-rosa-short} cluster. The workaround is to scale the worker nodes to allow the {product-title-short} components to be scheduled.
```
- Do the components schedule something, or are they scheduled? My revision assumes that they're scheduled -- feel free to adjust it if that's not the case.
